src/Components/Todo_list/Components/Columns.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import './Todo_list.css'
+import Tasks from './Tasks'
+import Data from '../TodoContext'
```

Comment:
Đặt tên `Data` không hợp lý, gây khó hiểu
Hãy giữ nguyên là 
```suggestion
import TodoContext from '../contexts/TodoContext'
```